install.sh

```diff
@@ -592,14 +592,20 @@ getshims() {
 
 killtree $({ set +x; } 2>/dev/null; getshims; set -x)
 
-do_unmount() {
+do_unmount_and_remove() {
     awk -v path="$1" '$2 ~ ("^" path) { print $2 }' /proc/self/mounts | sort -r | xargs -r -t -n 1 umount
+    awk -v path="$1" '$2 ~ ("^" path) { print $2 }' /proc/self/mounts | sort -r | xargs -r -t -n 1 umount rm -f 
```

Comment:
1. the command `umount rm -f` looks like a typo?
1. won't this never find anything because we're looking at /proc/self/mounts and we just unmounted everything?

```suggestion
    awk -v path="$1" '$2 ~ ("^" path) { print $2 }' /proc/self/mounts | sort -r | xargs -r -t -n 1 sh -c 'umount $0 && rm -f $0'
```

Reply:
In retrospect this appears to actually need to be `rm -rf`:

```
+ do_unmount_and_remove /var/lib/kubelet/pods
+ sort -r
+ awk -v path=/var/lib/kubelet/pods '$2 ~ ("^" path) { print $2 }' /proc/self/mounts
+ xargs -r -t -n 1 sh -c 'umount $0 && rm -f $0'
sh -c umount $0 && rm -f $0 /var/lib/kubelet/pods/de9c5e28-aebe-4ad5-b493-321b9931ce39/volumes/kubernetes.io~secret/default-token-78tkh
rm: cannot remove ‘/var/lib/kubelet/pods/de9c5e28-aebe-4ad5-b493-321b9931ce39/volumes/kubernetes.io~secret/default-token-78tkh’: Is a directory
sh -c umount $0 && rm -f $0 /var/lib/kubelet/pods/d40f0491-421e-4802-a7e0-4ee10351fc37/volumes/kubernetes.io~secret/metrics-server-token-b85k2
rm: cannot remove ‘/var/lib/kubelet/pods/d40f0491-421e-4802-a7e0-4ee10351fc37/volumes/kubernetes.io~secret/metrics-server-token-b85k2’: Is a directory
sh -c umount $0 && rm -f $0 /var/lib/kubelet/pods/9b0a0aae-bf26-4d93-a48b-3a331532a514/volumes/kubernetes.io~secret/local-path-provisioner-service-account-token-29fvt
rm: cannot remove ‘/var/lib/kubelet/pods/9b0a0aae-bf26-4d93-a48b-3a331532a514/volumes/kubernetes.io~secret/local-path-provisioner-service-account-token-29fvt’: Is a directory
sh -c umount $0 && rm -f $0 /var/lib/kubelet/pods/86af116e-eea2-4d31-9070-7789131f6289/volumes/kubernetes.io~secret/traefik-token-5kbks
rm: cannot remove ‘/var/lib/kubelet/pods/86af116e-eea2-4d31-9070-7789131f6289/volumes/kubernetes.io~secret/traefik-token-5kbks’: Is a directory
sh -c umount $0 && rm -f $0 /var/lib/kubelet/pods/86af116e-eea2-4d31-9070-7789131f6289/volumes/kubernetes.io~secret/ssl
rm: cannot remove ‘/var/lib/kubelet/pods/86af116e-eea2-4d31-9070-7789131f6289/volumes/kubernetes.io~secret/ssl’: Is a directory
sh -c umount $0 && rm -f $0 /var/lib/kubelet/pods/811ff188-5155-47d4-b4da-2fa4c85f4f99/volumes/kubernetes.io~secret/coredns-token-v9zjs
rm: cannot remove ‘/var/lib/kubelet/pods/811ff188-5155-47d4-b4da-2fa4c85f4f99/volumes/kubernetes.io~secret/coredns-token-v9zjs’: Is a directory
```